app/services/mapping/declaration/CurrencyExchangeBuilder.scala

```diff
@@ -29,8 +29,8 @@ import wco.datamodel.wco.dec_dms._2.Declaration.CurrencyExchange
 
 import scala.collection.JavaConverters._
 
-class CurrencyExchangeBuilder @Inject()() extends ModifyingBuilder[Declaration] {
```

Comment:
Whys the trait gone?

Reply:
as above "This was a rebase issue I had on Friday. looks like I resolved this file incorrectly will change it back"